privacyidea/lib/container.py

```diff
@@ -377,6 +377,26 @@ def get_all_containers(user: User = None, serial: str = None, ctype: str = None,
     return ret
 
 
+def container_page_generator(pagesize: int = 10, **kwargs) -> Generator[list[TokenContainerClass], None, None]:
```

Comment:
analogue to the token functions this should be something like `get_container_generator`.